test/e2e/user_management_test.go

```diff
@@ -556,66 +556,6 @@ func (s *userManagementTestSuite) TestUserDeactivation() {
 }
 
 func (s *userManagementTestSuite) TestUserBanning() {
-	s.Run("ban provisioned usersignup", func() {
-		hostAwait := s.Host()
-		memberAwait := s.Member1()
-		hostAwait.UpdateToolchainConfig(s.T(), testconfig.AutomaticApproval().Enabled(false))
-
-		// Create a new UserSignup and approve it manually
-		user := NewSignupRequest(s.Awaitilities).
-			Username("banprovisioned").
-			Email("banprovisioned@test.com").
-			ManuallyApprove().
-			TargetCluster(memberAwait).
-			RequireConditions(wait.ConditionSet(wait.Default(), wait.ApprovedByAdmin())...).
-			Execute(s.T())
-		userSignup := user.UserSignup
-
-		// Create the BannedUser
-		CreateBannedUser(s.T(), s.Host(), userSignup.Spec.IdentityClaims.Email)
-
-		// Confirm the user is banned
-		_, err := hostAwait.WithRetryOptions(wait.TimeoutOption(time.Second*15)).WaitForUserSignup(s.T(), userSignup.Name,
-			wait.UntilUserSignupHasConditions(wait.ConditionSet(wait.Default(), wait.ApprovedByAdmin(), wait.Banned())...))
-		require.NoError(s.T(), err)
-
-		// Confirm that a MasterUserRecord is deleted
-		_, err = hostAwait.WithRetryOptions(wait.TimeoutOption(time.Second*10)).WaitForMasterUserRecord(s.T(), userSignup.Spec.IdentityClaims.PreferredUsername)
-		require.Error(s.T(), err)
-		// confirm usersignup
-		_, err = hostAwait.WaitForUserSignup(s.T(), userSignup.Name,
-			wait.UntilUserSignupHasConditions(wait.ConditionSet(wait.Default(), wait.ApprovedByAdmin(), wait.Banned())...),
-			wait.UntilUserSignupHasStateLabel(toolchainv1alpha1.UserSignupStateLabelValueBanned))
-		require.NoError(s.T(), err)
-	})
-
-	s.Run("manually created usersignup with preexisting banneduser", func() {
-		hostAwait := s.Host()
-		memberAwait := s.Member1()
-		hostAwait.UpdateToolchainConfig(s.T(), testconfig.AutomaticApproval().Enabled(true))
-
-		id := uuid.Must(uuid.NewV4()).String()
-		email := "testuser" + id + "@test.com"
-		CreateBannedUser(s.T(), s.Host(), email)
-
-		// For this test, we don't want to create the UserSignup via the registration service (the next test does this)
-		// Instead, we want to confirm the behaviour when a UserSignup with a banned email address is created manually
-		userSignup := NewUserSignup(hostAwait.Namespace, "testuser"+id, email)
-		userSignup.Spec.TargetCluster = memberAwait.ClusterName
-
-		// Create the UserSignup via the Kubernetes API
-		err := hostAwait.CreateWithCleanup(s.T(), userSignup)
-		require.NoError(s.T(), err)
-		s.T().Logf("user signup '%s' created", userSignup.Name)
-
-		// Check the UserSignup is created and confirm that the user is banned
-		_, err = hostAwait.WaitForUserSignup(s.T(), userSignup.Name, wait.UntilUserSignupHasStateLabel(toolchainv1alpha1.UserSignupStateLabelValueBanned))
-		require.NoError(s.T(), err)
-
-		err = hostAwait.WaitUntilSpaceAndSpaceBindingsDeleted(s.T(), "testuser"+id)
-		require.NoError(s.T(), err)
-	})
-
```

Comment:
this didn't test a real scenario (we don't create UserSignups manually), thus it's better to remove it to not bring any confusion in the future.